src/generateRenderer/createSymbol.js

```diff
@@ -0,0 +1,26 @@
+const _ = require('lodash')
+
+module.exports = createSymbol
+
+function createSymbol (baseSymbol, color, type) {
+  const symbol = baseSymbol !== undefined ? _.cloneDeep(baseSymbol) : symbolTemplate(type) // TODO: figure out why cloneDeep is necessary
+  symbol.color = color
+  return symbol
+}
+
+function symbolTemplate (options) {
+  const renderers = {
```

Comment:
you typically don't want to define static variables inside function calls for performance reasons